app/javascript/packages/build-sass/cli.js

```diff
@@ -26,9 +26,8 @@ const { values: flags, positionals: fileArgs } = parseArgs({
   },
 });
 
-const isWatching = flags.watch;
-const outDir = flags['out-dir'];
-const loadPaths = [...flags['load-path'], ...getDefaultLoadPaths()];
+const { watch: isWatching, 'out-dir': outDir, 'load-path': loadPaths = [] } = flags;
+loadPaths.push(...getDefaultLoadPaths());
```

Comment:
The polyfill doesn't reflect that `flags['load-path']` may be `undefined`, which TypeScript flagged as not being possible to splat into an array from the prior logic:

```
app/javascript/packages/build-sass/cli.js:31:23 - error TS2488: Type 'string[] | undefined' must have a '[Symbol.iterator]()' method that returns an iterator.

31 const loadPaths = [...flags['load-path'], ...getDefaultLoadPaths()];
                         ~~~~~~~~~~~~~~~~~~
```

So effectively this is adding a default empty array. I could have inlined it, though I chose a bit of a rewrite to destructure the entire flags object and use that as the point at which to assign the default values.